src/Cake.SqlServer/Backup/RestoreSqlBackupSettings.cs

```diff
@@ -59,5 +72,13 @@ public RestoreSqlBackupSettings()
         /// </summary>
         public int? DifferentialBackupSetFile { get; set; }
 
+        
+    }
+
+    public enum DbUserMode
```

Comment:
Can you please add xml documentation with a brief explanation of what it does

Reply:
Done